app/components/embed/media_tag_component.rb

```diff
@@ -87,7 +87,27 @@ def enabled_streaming_sources
     def transcript
       return unless render_captions?
 
-      tag.track(src: file.vtt.file_url, kind: 'captions', srclang: 'en', label: 'English')
+      # A video clip may have multiple caption files in different languages.
+      # We want to enable the user to select from any of these options.
+      # We also want the different language options to be listed alphabetically.
+      safe_join(
+        file.vtt.sort_by { |cfile| caption_language(cfile.language) }.map do |caption_file|
+          lang_code = caption_file.language || 'en'
+          lang_label = caption_language(lang_code)
+          tag.track(src: caption_file.file_url, kind: 'captions', srclang: lang_code, label: lang_label)
+        end
+      )
+    end
+
+    def caption_language(language_code)
+      case language_code
```

Comment:
How about `LANGUAGE_LABELS.fetch(language_code, 'Caption')` instead of the `case`?

Reply:
This is a filler method so it's just intended to be temporary. Mike's working on a related issue so I'll wait to use his approach for getting labels based on language code.

Reply:
I am going to rewrite the filler method and leave a comment there that it should follow the caption code to label process from @mjgiarlo 's upcoming changes.